WebHostLib/stats.py

```diff
@@ -1,20 +1,24 @@
 from collections import Counter, defaultdict
-from itertools import cycle
+from colorsys import hsv_to_rgb
 from datetime import datetime, timedelta, date
 from math import tau
+import typing
 
 from bokeh.embed import components
-from bokeh.palettes import Dark2_8 as palette
+from bokeh.models import HoverTool
 from bokeh.plotting import figure, ColumnDataSource
 from bokeh.resources import INLINE
+from bokeh.colors import RGB
 from flask import render_template
 from pony.orm import select
 
 from . import app, cache
 from .models import Room
 
+PLOT_WIDTH = 600
```

Comment:
we may want to look into more responsive design, but this is probably more of a web design question.

Reply:
yeah, outside my area too, but in my opinion they look better like that... on my screen.
They support responsive but then the site also has to support that. Hopefully a webdesigner takes a shot at that at some point.